modules/oe_content_person/config/install/field.field.node.oe_person.oe_departments.yml

```diff
@@ -0,0 +1,34 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_departments
+    - node.type.oe_person
+  module:
+    - rdf_skos
+id: node.oe_person.oe_departments
+field_name: oe_departments
+entity_type: node
+bundle: oe_person
+label: 'Responsible department'
+description: ''
+required: false
+translatable: true
```

Comment:
This should be false. We are not making fields translatable from component, we just ensure that they can be set as translatable from the storage.